test/java/seedu/addressbook/storage/InvalidData.txt

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<AddressBook>
+    <persons>
+        <name>John Doe</name>
+
+        <email isPrivate="false">johnd@gmail.com</email>
+        <address isPrivate="false">John street, block 123, #01-01</address>
+    </persons>
+</AddressBook>
```

Comment:
In the `StorageFile` class, it also only converts directly from data to the relevant Object as long as the xml structure is correct